src/styles/theme-data.css

```
@@ -61,7 +61,9 @@
   --spacing-header-height: 4.5rem;
   --spacing-sidebar-width: var(--sidebar-width);
   --spacing-mobile-sidebar-height: calc(
-    100svh - var(--spacing-header-height) - 2rem
+    100svh -
```

Stylelint (17.14.0)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)